### ISSUE TITLE
Does not work chunked loading in Chrome

### ISSUE THREAD

**Issue by iberezansky** [mozilla/pdf.js#9717]
Attach (recommended) or Link to PDF file here:
http://wp.iberezansky.net/wp-content/uploads/custom/SMC-2017-catalogue.pdf

Configuration:
- Web browser and its version:  Chrome x64 66.0.3359.139
- Operating system and its version: Windows 10 Pro x64 
- PDF.js version: 2.0.493
- Is a browser extension: no

Steps to reproduce the problem:
1. Clear browser cache
2. Open http://wp.iberezansky.net/?page_id=60

What is the expected behavior? (add screenshot)
The library has to load data only for retrieved pages 1 and 2. Loading progress has to go to ~20%. In Firefox and IE11 works fine.

What went wrong? (add screenshot)
The library loads whole PDF 100%.

Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension):
http://wp.iberezansky.net/?page_id=60

**Comment by sirisian** [comment]
Yeah, this seems really easy to reproduce. Is there a simple way to disable chunked loading or something in the meantime? It seems like there are tons of bugs related to it that have been closed in the past. (Supposedly this is fixed in 67 of chrome).
```
Uncaught (in promise) Error: Bad end offset: 275384
    at ChunkedStream_onReceiveData [as onReceiveData] (pdf.worker.js:14653)
    at ChunkedStreamManager_onReceiveData [as onReceiveData] (pdf.worker.js:14998)
    at pdf.worker.js:14877
```

**Comment by Snuffleupagus** [comment]
Possibly a duplicate of issue #9544, which according to https://github.com/mozilla/pdf.js/issues/9544#issuecomment-374176371 was a (now fixed) upstream bug in Google Chrome.
